src/app/account-management/account/account.component.ts

```diff
@@ -144,9 +144,10 @@ export class AccountComponent implements OnInit {
   }
 
 
+  /*
```

Comment:
Remove commented code if not planning on adding back or referencing later. Keep if so.